posts/2020-09-29-microprofile-openapi-beta-200011.adoc

```diff
@@ -0,0 +1,253 @@
+   "description": "Milliseconds since January 1, 1970, 00:00:00 GMT" \
+ }
+----
+
+Enable OpenAPI 2.0 in the server.xml, along with any other features you’re using.
+
+[source, xml]
+----
+<featureManager>
+       <feature>mpOpenAPI-2.0</feature>
+   </featureManager>
+----
+
+For more information:
+
+* link:{url-prefix}/guides/microprofile-openapi.html[Guide: Documenting RESTful APIs]
+* link:https://download.eclipse.org/microprofile/microprofile-open-api-2.0-RC3/apidocs/[Javadoc]
+* link:https://download.eclipse.org/microprofile/microprofile-open-api-2.0-RC3/microprofile-openapi-spec.html#release_notes_20[Release notes]
+
+=== Try it now
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 14, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>20.0.0.11-beta</version>
+  <type>pom</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[20.0.0.11-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+image::img/blog/logos/Jakarta_EE_logo.png[ Jakarta EE Logo,width=70%,align="center"]
+
+The main change you should be aware of in Jakarta EE 9 planned release is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have more Jakarta EE 9 features with their name change completed. In this release most features had improvments, including many moving up to the latest API Release Candidates.
+
+This Open Liberty beta introduces the following Jakarta EE 9 features which now possess their all-new Jakarta EE 9 package names:
+
+* Jakarta Enterprise Beans Remote 4.0 (`ejbRemote-4.0`)
+* Jakarta EE Application Client 9.0 (`javaeeClient-9.0`) (Now supports Jakarta Enterprise Beans Remote Client 4.0)
+
+
+These join the Jakarta EE 9 features in link:{url-about}/blog/?search=beta&key=tag[previous Open Liberty betas]:
+
+* Jakarta Authentication 2.0 (`jaspic-2.0`)
+* Jakarta Authorization 2.0 (`jacc-2.0`)
+* Jakarta Persistence 3.0 (includes Eclipselink 3.0-RC1.) (`jpa-3.0`)
+* Jakarta XML Binding 3.0 (`jaxb-3.0`)
+* Jakarta Managed Beans 2.0 (`managedBeans-2.0`)
+* Jakarta Concurrency 2.0 (`concurrent-2.0`)
+* Jakarta Enterprise Beans Home 4.0 (`ejbHome-4.0`)
+* Jakarta Enterprise Beans Lite 4.0 (`ejbLite-4.0`)
+* Jakarta Bean Validation 3.0 (`beanValidation-3.0`)
+* Jakarta Contexts and Dependency Injection 3.0 (`cdi-3.0`)
+* Jakarta WebSocket 2.0 (`websocket-2.0`; currently the integration with CDI is not completed)
```

Comment:
"completed" -> "complete" (and update the template)

Reply:
Done

Reply:
This isn't included in the template